assets/dns/daemonset.yaml

```diff
@@ -27,6 +27,16 @@ spec:
         - containerPort: 9153
           name: metrics
           protocol: TCP
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 10
```

Comment:
Might be a little conservative — normal startup of the coredns process is sub-second. Not sure whether we should be shaving seconds here.